stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -210,8 +210,18 @@ module SwiftGlibc [system] {
       export *
     }
 % end
+% if CMAKE_SDK == "ANDROID":
+    module cpio {
```

Comment:
Why is `cpio` being added?

Reply:
In this case, the previous block (from L191 to L212) is for Linux and FreeBSD, and includes `aio.h`, `cpio.h`, `fmtmsg.h`, `nl_types.h` and `ulimit.h`. From those, the only ones presents in the NDK (r18) are `cpio.h` and `nl_types.h`. I added a separate block because it is something that happens in some cases in this file, and it seems easier to read that the bigger block with a lot of exceptions.

Reply:
Bleh.  I don't see a nice way to handle this.